gravitee-apim-gateway/gravitee-apim-gateway-policy/src/main/java/io/gravitee/gateway/policy/impl/PolicyFactoryCreatorImpl.java

```diff
@@ -27,20 +28,20 @@
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-public class PolicyFactoryCreator implements FactoryBean<PolicyFactory> {
+public class PolicyFactoryCreatorImpl implements PolicyFactoryCreator {
 
-    private final Logger logger = LoggerFactory.getLogger(PolicyFactoryCreator.class);
+    private final Logger logger = LoggerFactory.getLogger(PolicyFactoryCreatorImpl.class);
 
     private final Environment environment;
     private final PolicyPluginFactory policyPluginFactory;
 
-    public PolicyFactoryCreator(final Environment environment, final PolicyPluginFactory policyPluginFactory) {
+    public PolicyFactoryCreatorImpl(final Environment environment, final PolicyPluginFactory policyPluginFactory) {
```

Comment:
Can you please rely on the io.gravitee.node.api.Configuration instead of Environment ?

Reply:
sure !